Assets/GameAssets/Player/PlayerScripts/PlayerInteraction.cs

```diff
@@ -44,7 +44,7 @@ private void CheckForInteractable()
         SetRaycast();
         if (Physics.Raycast(_ray, out RaycastHit hit, _maxReach))
         {
-            GiveInteractableFeedback(hit.collider.gameObject);
+            //GiveInteractableFeedback(hit.collider.gameObject);
```

Comment:
If this isn't being used it should be deleted, otherwise it should be uncommented

Reply:
^